easybuild/easyconfigs/n/networkx/networkx-1.9.1-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'networkx'
+version = '1.9.1'
+
+homepage = 'https://pypi.python.org/pypi/networkx'
+description = """NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,
+ and functions of complex networks."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
```

Comment:
maybe we should start looking into simplifying this for `PythonPackage` easyblock? In principle, only the python version should be specified.